db/db_test2.cc

```diff
@@ -5891,13 +5891,59 @@ TEST_F(DBTest2, BottommostTemperature) {
   ASSERT_OK(Flush());
   ASSERT_OK(dbfull()->TEST_WaitForCompact());
 
-  Reopen(options);
```

Comment:
Why is it removed? The point is to test that the temperature info is correctly persistented to manifest and recover back.